File: 프로그래머스/Lv.0/최댓값 만들기 (2)/120862.java
class Solution {
    public int solution(int[] numbers) {
        int answer = 0;
        
        int max = numbers[0] * numbers[1];
        for (int i=0; i<numbers.length-1; i++) {
            for (int j=i+1; j<numbers.length; j++) {
                max = Math.max(max, numbers[i] * numbers[j]);
            }
        }
        
        answer = max;
        
        return answer;
    }
}

File: 프로그래머스/Lv.0/최댓값 만들기 (2)/120862.js
function solution(numbers) {
    let answer = 0;
    
    numbers.sort((a, b) => a - b);
    
    const len = numbers.length - 1;
    answer = Math.max(numbers[0] * numbers[1], numbers[len] * numbers[len-1]);
    
    return answer;
}
